pkg/server/smtp/listener.go

```diff
@@ -133,6 +138,8 @@ func (s *Server) serve(ctx context.Context) {
 	var tempDelay time.Duration
 	for sessionID := 1; ; sessionID++ {
 		if conn, err := s.listener.Accept(); err != nil {
+			slog := log.With().Str("proxy", "false").Logger()
```

Comment:
Hard coded false does not look useful?

Reply:
yes that can go away - it was really code used for debugging